airbyte-cdk/python/unit_tests/sources/declarative/requesters/error_handlers/test_composite_error_handler.py

```diff
@@ -16,76 +18,76 @@
     "test_name, first_handler_behavior, second_handler_behavior, expected_behavior",
     [
         (
-            "test_chain_retrier_ok_ok",
-            response_status.SUCCESS,
-            response_status.SUCCESS,
-            response_status.SUCCESS,
+                "test_chain_retrier_ok_ok",
```

Comment:
nit: This double indenting doesn't seem quite right. Did your precommit linter do this?